tools/ovc/CMakeLists.txt

```diff
@@ -35,6 +35,13 @@ install(DIRECTORY ${OpenVINOConverter_SOURCE_DIR}/openvino
         ${OV_CPACK_COMP_OVC_EXCLUDE_ALL}
         USE_SOURCE_PERMISSIONS)
 
+if(ENABLE_TESTS)
+    install(DIRECTORY unit_tests
+            DESTINATION tests/ovc
+            COMPONENT tests
+            EXCLUDE_FROM_ALL)
+endif()
```

Comment:
why is it needed in current PR? seems unrelated to namespaces work

Reply:
It is a leftover from previous tests where OVC tests were copied to avoid being run from same directory. Similar tactic to MO tests.

Would you like to remove it and test it over with current approach?

Reply:
Reverted and works.